internal/resource/role/model.go

```diff
@@ -0,0 +1,67 @@
+package role
+
+import (
+	"context"
+
+	sgsdkgo "github.com/StackGuardian/sg-sdk-go"
+	flatteners "github.com/StackGuardian/terraform-provider-stackguardian/internal/flattners"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type RoleAllowedPermissionsPathsModel []string
```

Comment:
Cannot be []string needs to be []types.String
Then you'll need to iterate over []types.String value to convert it to []string